crates/vm/levm/Cargo.toml

```diff
@@ -43,6 +43,7 @@ strum = { version = "0.27.1", features = ["derive"] }
 k256.workspace = true
 substrate-bn = { version = "0.6.0", optional = true }
 secp256k1 = { workspace = true, optional = true }
+ziskos = { git = "https://github.com/0xPolygonHermez/zisk.git", tag = "v0.14.0", optional = true }
```

Comment:
Let's put this a bit spaced from the other deps, with a comment mentioning zisk, to make it easier to see it's a proving-only dependency